snc-library.sh

```diff
@@ -265,6 +266,7 @@ function wait_till_cluster_stable() {
     local retryCount=30
     local numConsecutive=3
     local count=0
+    local ignoreNamespace=${1:-"none"}
```

Comment:
Not sure the `none` default value will do the right thing? maybe we can just ignore it?

Reply:
@cfergeau it is doing as expected until there is some default namespace named as `none` appear which is highly a case.

Reply:
I have no idea what is expected with `none` :) a failure ?